.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+### Before submitting
+
+Please complete the following checklist when submitting a PR:
+
+- [ ] All new features must include a unit test.
+      If you've fixed a bug or added code that should be tested, add a test to the
+      [`tests/`](tests/) directory!
+
+- [ ] All new functions and code must be clearly commented and documented.
+      If you do make documentation changes, make sure that the docs build and
+      render correctly by running `make docs`.
+
+- [ ] Ensure that the test suite passes, by running `make test`.
+
+- [ ] Add a new entry to the `.github/CHANGELOG.md` file, summarizing the
+      change, and including a link back to the PR.
+
+- [ ] Ensure that code is properly formatted by running `make format`. 
```

Comment:
Is this a TODO to add to the makefile?

Reply:
Looks like this was not added in my previous push to the #108 branch. That is now fixed.